frontend/src/main/utils/useBackend.js

```diff
@@ -63,8 +63,8 @@ export function useBackendMutation(
   const queryClient = useQueryClient();
 
   return useMutation((object) => wrappedParams(objectToAxiosParams(object)), {
-    onError: (error) => {
-      const errorMessage = `useBackendMutation: Error communicating with backend via ${error.response.config.method} on ${error.response.config.url}`;
+    onError: (_error) => {
```

Comment:
Do you realize what you've done here?

Every single backend action that has ANY error AT ALL in the ENTIRE app is now going to use this error message.

This is not an acceptable way to solve this.   It "seems" to work, but there are may other possible errors.  You need a much more surgical approach.

This isn't it.